src/website/views/components/toolbar/_properties.pug

```diff
@@ -1,6 +1,34 @@
 h2 Chi Vue
 h3 Props
 
+h4 Toolbar 
+section.chi-table.chi-table__options.-bordered.-my--3 
+  div(style='overflow-x:auto;')
+    table.-text(cellpadding='0' cellspacing='0')
+      thead 
+        tr 
+          th
+            div Property
```

Comment:
Is the div necessary here?
```suggestion
            | Property
```

Reply:
The same for all the following cases, please

Reply:
Thing is it's structured like this in most pages, but I'll only remove them for toolbar/ to avoid a big PR